docs/product/roadmap.md

```diff
@@ -0,0 +1,65 @@
+
+## Product Vision
+* Fully transition the FAC PMO from Census to TTS
+* Transition the [current Census FAC product](https://facweb.census.gov/) into something TTS can maintain
+* Reduce risk and prove value by delivering incremental product updates 
+
+## Release: MVP
+**Goals:** Launch a new FAC intake experience for at least some of current FAC customers on or around 10/1 to begin accepting audits for FY2022. Per the discussed and approved [migration plan](https://github.com/GSA-TTS/FAC/blob/main/docs/product/decisions/0004-P-migration-plan.md), the idea is to keep the old Census FAC running and import FY2022 submissions into that for data disseminination. This will help limit scope and reduce risk of the MVP. 
+
+**Success Metrics (proposed):** 
+* Number of successful FY2022 submissions compared to previous time frame
+* Number of support inquiries compared to previous time frame
+
+**Key Features:**
+* ✅ Technical foundation / infrastructure
+* (80% ✅) Create a new audit (pre SAC forms)
+* (in design) Submit an audit package (post SAC forms)
+* Excel file ingest
+* Login.gov integration
+* Homepage with nav linking to old FAC static pages + 
+* Minimal new content living on new site (Informational content for new system features)
+* Email notifications (status change notifications, etc)
+* “Frontend” user roles + UI 
+    * These are the users who submit/certify audits
+    * Let the person who creates an audit add other users with the same perms. 
+    * Subset of folks allowed to certify.
+* “Backend” user roles + UI 
+    * These are admins and customer support roles
+    * DJango admin for UI?
+* Have a path to transfer data from new FAC to current FAC, preferably in real time
+* ATO
+* Release (domain name, public launch, user relations)
+    * Unknowns here for cross agency coordination
+
+## Possibly not MVP release
+
+**Key Features:**
+* Automated post-submission verification
+    * Very basic, like year in excel match year submitted? UEI’s match?
+* Non-xls input to audit packages
+* Copying over FAQ’s and other pages
+* Submitting revisions to an audit package
+
+## Definitely not MVP release
+
+**Goals:** Fully replace the Census FAC, which at this point can be taken off the internet. At this point the new TTS FAC will be the authoratative source for all historical FAC data. 
+
+**Success Metrics (proposed):** 
+* Amount of FAC data disseminated compared to previous time frame
+* Number of support inquiries compared to previous time frame
+
+**Key Features:**
+* Data dissemination (exports, full API, etc)
+* Search
+* Handling disseminating public vs/ non-public data
+* Manual quality assurance of audit submissions
```

Comment:
I don't think GSA should be in the business of doing manual quality reviews ever. So I'd remove this.

Reply:
Removed